mr_project/mr_app/__init__.py

```diff
@@ -0,0 +1,8 @@
+from neo4j import GraphDatabase
+
+uri = 'bolt://localhost:7687'
+
+DRIVER = GraphDatabase.driver(uri, auth=('neo4j', 'password'))
```

Comment:
Hard coded credentials should be replaced with environment variables. We also need to rethink when to initialize the neo4j client because with the current setup, we can't run any unit tests unless we can reach the neo4j database. Unit tests should be able to run without any external resources

Reply:
this means putting initialization info into mr_env?

Reply:
Not exactly. Environment variables are key-value pairs and `mr_env` just contains the values used for testing. The environment variables are accessed from python using the [`os.environ`](https://docs.python.org/3/library/os.html#os.environ) method. So yes, you should add the test values of the new environment variables to `mr_env`, but you'll also need to add logic to access them.